wgengine/monitor/monitor.go

```diff
@@ -282,6 +286,10 @@ func (m *Mon) notifyRuleDeleted(rdm ipRuleDeletedMessage) {
 	}
 }
 
+func (m *Mon) useInterestingInterfaces(i interfaces.Interface, ips []netaddr.IPPrefix) bool {
```

Comment:
docs on this method now.

Or just call it `isInterfaceInteresting` and clarify that `ips` is its current IPs.